api/tests/Feature/SmokeTest.php

```diff
@@ -2,18 +2,10 @@
 
 declare(strict_types=1);
 
-use App\Entity\User;
-
-use function Eerison\PestPluginApiPlatform\{
-    get,
-    findIriBy,
-    assertResponseIsSuccessful
-};
-
-it('is returning a successful response', function (string $url): void {
+it('is returning a successful response', static function (string $url): void {
```

Comment:
You don't need to use anonymous function as static,

If it was some rule that you added please remove it :)